wallets/provider-walletconnect-2/src/index.ts

```diff
@@ -176,26 +163,33 @@ export const switchNetwork: SwitchNetwork = async ({
   instance,
   meta,
   getState,
+  updateChainId,
 }) => {
-  const needRecreateSession = needSessionRecreateOnSwitchNetwork(instance);
-  config.isAsyncSwitchNetwork = true;
-  if (needRecreateSession) {
-    config.isAsyncSwitchNetwork = false;
-    /**
-     * In case of trust wallet that doesn't support switch chain method,
-     * we need to handle it manually by deleting last session and create a new one
-     * with correct chain id.
-     */
-    const session = await trySwitchByCreatingNewSession(instance, {
-      network,
-      meta,
-    });
-    instance.session = session;
+  const evm = evmBlockchains(meta);
+  const chainId = evm.find((chain) => chain.name === network)?.chainId || '0x1';
```

Comment:
Why we are falling back to `0x1` when we couldn't match anything? it shouldn't be throwing an error?

Reply:
Thanks. Could you also send it to our `logging`? I mean use `logError` as well.